rest/v9/index.ts

```diff
@@ -435,9 +435,19 @@ export const Routes = {
 		return parts.join('/') as `/channels/${Snowflake}/threads` | `/channels/${Snowflake}/messages/${Snowflake}/threads`;
 	},
 
+	/**
+	 * Route for:
+	 * - GET `/guilds/{guild.id}/threads/active`
+	 */
+	guildActiveThreads(guildId: Snowflake) {
+		return `/guilds/${guildId}/threads/active` as const;
+	},
+
 	/**
 	 * Route for:
 	 * - GET `/channels/{channel.id}/threads/active`
+	 * 	 (deprecated, use [List Active Guild Threads](https://discord.com/developers/docs/resources/guild#list-active-threads) instead.
```

Comment:
Use `@deprecated` instead

Reply:
didnt seem to render inline, and that might mark the whole function as deprecated while its only deprecated for that particular route